### ISSUE TITLE
is there a specific

### ISSUE THREAD

**Issue by lord-dubious** [WillYochim/HyperLiquidFarm#1]
is there a specific token this bot trades

**Comment by WillYochim** [comment]
Currently hard coded for BTC and ETH. You can trade anything you want though inside the function generate_trade_data()

**Comment by lord-dubious** [comment]
is it fully auntonomus? it just swaps maintaing the exact value?

**Comment by WillYochim** [comment]
Yes and there is a predefined value array you can change based on your preferred trade size range. It should keep running by itself cycling through forever. As always verify the logic checks out and do testing first.

**Comment by lord-dubious** [comment]
please can i pm or dm u and ask some questions
